src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -24,6 +24,38 @@
 logger = get_logger(__name__)
 
 
+def set_legal_hold(cmd, client, container_name, account_name, tags, resource_group_name=None):
+    LegalHold = cmd.get_models('LegalHold', resource_type=ResourceType.MGMT_STORAGE)
+    legal_hold = LegalHold(tags=tags)
+    return client.set_legal_hold(resource_group_name, account_name, container_name, legal_hold)
```

Comment:
avoid using positional arguments. please fix in next PR.

Reply:
will fix it in next PR.

---
In reply to: [521918599](https://github.com/Azure/azure-cli/pull/15845#discussion_r521918599) [](ancestors = 521918599)